acceptance/lib/helper.rb

```diff
@@ -34,7 +34,7 @@ def self.initialize_config(options)
     puppet_build_version = get_option_value(options[:puppet_build_version],
                          nil, "Puppet Agent Development Build Version",
                          "PUPPET_BUILD_VERSION",
-                         "b04c9f4a8d4eace424a718073315a7da0a34f3b5",
+                         "2aa05621ebb25ce724760ab4e65425b8a55f3f60",
```

Comment:
This is supposed to be the puppet-agent commit matching `1.10.0.337.g2aa0562` right?